docker/observability/services/prometheus.yml

```diff
@@ -1,15 +1,22 @@
+x-common-extra-hosts: &common-extra-hosts
+  extra_hosts:
+    - 'agora-gene-api:host-gateway'
+
 services:
   observability-prometheus:
-    image: ghcr.io/sage-bionetworks/observability-prometheus:${AMP_ALS_VERSION:-local}
+    <<: *common-extra-hosts
```

Comment:
This demonstrates that YAML aliases can be used within Docker Compose files. In a subsequent PR, I will explore defining the alias in a separate file to enable its reuse across multiple files. The goal is to centralize the alias definition for better maintainability.